rpcd/playbooks/roles/rpc_maas/templates/filebeat_process_check.yaml.j2

```diff
@@ -0,0 +1,19 @@
+type: agent.plugin
+label: "filebeat_process_check--{{ ansible_hostname }}"
+disabled    : false
+period      : "{{ maas_check_period }}"
+timeout     : "{{ maas_check_timeout }}"
+details     :
+    file    : run_plugin_in_venv.sh
+    args    : ["{{ maas_plugin_dir }}process_check.py", {% if inventory_hostname in groups['all_containers'] %} "-c", "{{ inventory_hostname }}" {% else %}{% endif %}"{{ filebeat_process_names|join(",") }}"]
```

Comment:
You need `"{{ inventory_hostname }}", ` because this is presently invalid YAML. You got this right in the elasticsearch check but you also need to update the rsyslog check.